package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "name-your-contributors",
-  "version": "3.8.3",
+  "version": "3.9.0",
```

Comment:
@RichardLitt Do you think it's worth manually bumping the version (to say `3.10.0`), or will TravisCI (assuming the project wasn't deactivated since they changed how they handle FOSS projects) pick it up and adequately publish it to NPM (which is seating on `v3.8.3` as of now)?

Reply:
Manually pumped. Those sound like good things to add, too.